ballerina/message_producer.bal

```diff
@@ -69,6 +69,13 @@ public isolated client class MessageProducer {
 };
 
 isolated function getJmsMessage(Session session, Message message) returns handle|Error {
+    handle jmsMessage = check constructJmsMessage(session, message);
+    check updateReplyToMessageField(session, jmsMessage, message.replyTo);
+    check updateCorrelationIdField(jmsMessage, message.correlationId);
+    return jmsMessage;
+}
+
+isolated function constructJmsMessage(Session session, Message message) returns handle|Error {
     if message is TextMessage {
         handle jmsMessage = check session.createJmsMessage("TEXT");
```

Comment:
shall we have a constant here?

Reply:
Fixed with [a5870abda2f378fbd1ebdc6334d6d3d5adf390f5](https://github.com/ballerina-platform/module-ballerinax-java.jms/pull/82/commits/a5870abda2f378fbd1ebdc6334d6d3d5adf390f5)